### PR TITLE
Tidy up invoker based tests

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

**Comment by tfennelly** [comment]
:+1:

File: pom.xml [hunk 1]
```diff
@@ -265,6 +263,44 @@
...
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>mrm-maven-plugin</artifactId>
+        <version>1.0-beta-2</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>start</goal>
```
Comment (aheritier) [inline pom.xml]: @stephenc It is probably useless to start/stop mrm outside of the run-its profile ?

Comment (stephenc) [inline pom.xml]: good call

**Comment by aheritier** [comment]
:+1:

**Pull request merged**